config/config.go

```diff
@@ -151,6 +151,24 @@ func BuildConfig(v *viper.Viper) (Config, bool, error) {
 		return Config{}, false, fmt.Errorf("failed to unmarshal source subnets: %w", err)
 	}
 
+	// Explicitly overwrite the configured KMS configuration
```

Comment:
Should we support setting different KMS configurations for each supported destination chain via env vars, similarly to how we do the raw private keys?

Reply:
I think we should hold off on more advanced/fine grained KMS integration until we have explicit requests for such features. I imagine we'll want a mechanism to select different keys for different purposes or at different times, but I'm not quite sure what that would be. For instance, I know round-robin cycling through a set of keys is a common technique.

Reply:
Discussed some more offline, and concluded that the global KMS option is unnecessarily complicated. The KMS settings are not sensitive, so they can safely be provided via config file.